### PR TITLE
Add the GenomeSet to the objects created 

### PR DESCRIPTION
**Pull request by landml**
when doing Multiple Assemblies or Multiple Genomes. PTV-1202
@JamesJeffryes

### CODE REVIEW

File: lib/RAST_SDK/RAST_SDKImpl.pm [hunk 1]
```diff
@@ -1540,18 +1540,26 @@ sub annotate_genomes
...
+    if (defined $params->{output_genome} && $params->{output_genome} gt ' ') {
+        my $output_genomeset = $params->{output_genome};
+        my $report_reference = Bio::KBase::kbaseenv::su_client()->KButil_Build_GenomeSet({
+            workspace_name => $params->{workspace},
+            input_refs => $output_genomes,
+            output_name => $output_genomeset,
+            desc => 'GenomeSet Description'
+        });
+
+        my $newset = Bio::KBase::kbaseenv::get_objects([{
```
Comment (JamesJeffryes) [inline lib/RAST_SDK/RAST_SDKImpl.pm]: Ideally, I'd use get_object_info3 or use the `no_data` option but I suppose genome sets are pretty small

File: lib/RAST_SDK/RAST_SDKImpl.pm [hunk 2]
```diff
@@ -1540,18 +1540,26 @@ sub annotate_genomes
...
+            workspace => $params->{workspace},
+            name => $output_genomeset
+        }]);
+        my $info = $newset->[0]->{info};
+        Bio::KBase::kbaseenv::add_object_created({
+            "ref" => $info->[6]."/".$info->[0]."/".$info->[4],
```
Comment (JamesJeffryes) [inline lib/RAST_SDK/RAST_SDKImpl.pm]: You can probably actually get away with `$params->{workspace}."/".$output_genomeset and avoid looking up the object entirely. These refs are converted to UPAs when saved in the WS

Comment (landml) [reply lib/RAST_SDK/RAST_SDKImpl.pm]: Where so you get the documentation for this? I'm just copying other code that I've seen. I don't know where to find the no_data option.

**Comment by JamesJeffryes** [comment]
So turns out that no_data is actually an option for get_objects2:
https://ci.kbase.us/services/ws/docs/Workspace.html#funcdefWorkspace.get_objects2

**Comment by landml** [comment]
Okay, I made the change.

**Pull request merged**